os/board/rtl8730e/src/rtl8730e_st7789.c

```diff
@@ -31,9 +31,13 @@
 #define CONFIG_LCD_ST7789_SPI_PORT 1
 #endif
 
+#if CONFIG_BOARD_REVISION_VERSION == 5
+#error "ERROR, PA9 and PA10 pins are duplicated with the UART pins."
+#else
```

Comment:
What does this mean? Rev 5 can't use the ST7789 LCD?

Reply:
Currently, the rev 5 board cannot use PA9 PA10 in st7789.

This is to prevent malfunctions on the rev 5 board when the mistake st7789 is enabled.

However, if your opinion is a better way to arbitrarily set the pins you don't use, I'll change this.